rasa/core/train.py

```diff
@@ -70,6 +70,44 @@ async def train(
     return agent
 
 
+async def save_fingerprint(
+    story_file_path: Text,
```

Comment:
what about the other data files e.g. `nlu.yaml`?

Reply:
The existing fingerprint function uses the `TrainingDataImporter` to get all the rest, and story file, domain file, and config are required only to load the importer. 

For more details, see 
1) `model_fingerprint` function in `rasa.model` 
2) `load_from_config` in `rasa.shared.importers.importer`

Reply:
yeah but as far as I am aware `TrainingDataImporter` will only be able to provide `get_nlu_data()` if it was passed an NLU data file to `load_from_config`.

Reply:
That's interesting because the existing fingerprint command doesn't use the NLU data, and the importer that's currently used for fingerprints is always the `CoreDataImporter`. 

But you might be true, and it actually should. Let's return to this once we finish discussing what to do with this command :)